packages/expo-modules-core/ios/NativeModulesProxy/EXNativeModulesProxy.mm

```diff
@@ -414,8 +414,6 @@ - (void)installExpoTurboModules
 
     [EXJavaScriptRuntimeManager installExpoModulesToRuntime:runtime withSwiftInterop:_swiftInteropBridge];
     [_swiftInteropBridge setRuntime:runtime];
-
-    expo::installRuntimeObjects(*jsiRuntime, _bridge.jsCallInvoker, self);
```

Comment:
nit: Maybe rename this method (`installExpoTurboModules`) as it is now unrelated to TurboModules.